test/units/testsuite-60.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -eux
+set -o pipefail
+
+systemd-analyze log-level debug
+systemd-analyze log-target journal
+
+NUM_DIRS=20
+
+# mount/unmount enough times to trigger the /proc/self/mountinfo parsing rate limiting
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    mkdir "/tmp/meow${i}"
+done
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    mount -t tmpfs tmpfs "/tmp/meow${i}"
+done
+
+systemctl daemon-reload
+systemctl list-units -t mount tmp-meow* | grep -q tmp-meow
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    umount "/tmp/meow${i}"
+done
+
+# figure out if we have entered the rate limit state
+
+exited_rl=0
+timeout="$(date -ud "2 minutes" +%s)"
+while [[ $(date -u +%s) -le ${timeout} ]]; do
```

Comment:
Hmm, I guess this should be `(( ... ))` instead of `[[ ... ]]`, as here we compare numbers instead of string.
`[[ ... ]]` compares the strings lexicographically. I am not sure about the lexicographical order of e.g. "99" vs "100"...

Reply:
Oh, `<` and `>` are lexicographic, and `-le` and friends are arithmetic comparison... Hmm... I did not know that. Thanks.